pr-data.csv

```diff
@@ -1739,7 +1739,7 @@ https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar
 https://github.com/apache/nifi,41ff6f076b2e5346cf695ff2064e5dcfe249c124,nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core,org.apache.nifi.controller.TestEncryptedFileSystemSwapManager.testSwapOutSwapIn,OD,,,
 https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core,org.apache.nifi.controller.TestFlowController.testSynchronizeFlowWhenExistingMissingComponentsAreDifferent,ID,,,
 https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core,org.apache.nifi.controller.TestFlowController.testSynchronizeFlowWhenProposedMissingComponentsAreDifferent,ID,,,
-https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-standard-services/nifi-distributed-cache-services-bundle/nifi-distributed-cache-server,org.apache.nifi.distributed.cache.server.map.DistributedMapCacheTest.testSubMap,ID,,,
+https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-standard-services/nifi-distributed-cache-services-bundle/nifi-distributed-cache-server,org.apache.nifi.distributed.cache.server.map.DistributedMapCacheTest.testSubMap,ID,Accepted,https://github.com/apache/nifi/pull/6486,
```

Comment:
The PR https://github.com/snowflakedb/snowflake-jdbc/pull/1164 should be counted as `InspiredAFix`, not `Accepted`.

Reply:
@xieyt2000 Can I know why? It seems like the developer directly merged my PR as `sfc-gh-igarish merged 2 commits into snowflakedb:master from yannizhou05:master yesterday`.

Reply:
Sorry I pasted the wrong link. Should be https://github.com/apache/nifi/pull/6486 (The PR in the line I commented)

Reply:
The developer mentioned `fixed the checkstyle issue and merged`, and the codes merged were exactly the same as my code. I thought InspiredAFix meant the developer was inspired to fix tests by their own codes?

Reply:
You're right, it seems that it's a common practice in NiFi to close and merge by committing.
@darko-marinov, should we count that as `Accepted` or `InspiredAFix`?

Reply:
Fine to count as `Accepted` when it's exactly the same code, even if it was not `Merged` via the GitHub interface but committed+pushed to the repo in another way.